websites/G/GG App/presence.ts

```diff
@@ -0,0 +1,134 @@
+const presence = new Presence({
+    clientId: "928868301105426463"
+  }),
+  browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+  const presenceData: PresenceData = {
+      largeImageKey: "lg",
+      startTimestamp: browsingTimestamp
+    },
+    path = document.location.pathname,
+    [timestamps, cover, buttons] = await Promise.all([
+      presence.getSetting<boolean>("timestamps"),
+      presence.getSetting<boolean>("cover"),
+      presence.getSetting<boolean>("buttons")
+    ]);
+  // Apparently "/games" is a user page so .pathname can't be used to detect a game but instead the header
+  if (
+    document.querySelector<HTMLHeadingElement>(
+      "#root > div > div.css-1dbjc4n > div > div.w-full.xl\\:max-w-max.mx-auto.-mt-64.sm\\:-mt-48.px-4 > div.flex.flex-col.md\\:flex-row.pt-8 > div.w-full.md\\:w-3\\/4.md\\:pl-4.lg\\:pl-8 > div:nth-child(1) > h1"
```

Comment:
```suggestion
      "#root > div > div.css-1dbjc4n > div > div > div > div > div:nth-child(1) > h1"
```
It should still be the same. For other selectors as well

Reply:
Will fix ASAP